applications/RomApplication/CMakeLists.txt

```diff
@@ -5,7 +5,41 @@ message("**** configuring KratosRomApplication ****")
 ################### PYBIND11
 include(pybind11Tools)
 
-include_directories( ${KRATOS_SOURCE_DIR}/kratos )
+add_definitions( -DEIGEN_MPL2_ONLY)
+
+include_directories(
+    ${KRATOS_SOURCE_DIR}/kratos
+    SYSTEM ${KRATOS_SOURCE_DIR}/applications/LinearSolversApplication/external_libraries/eigen3
+    SYSTEM ${KRATOS_SOURCE_DIR}/applications/LinearSolversApplication/
+)
+
+if( USE_EIGEN_MKL MATCHES ON )

```

Comment:
I strongly recommend to avoid this

if you need eigen, then you should make the romapp dependent on the LinearSolversApp with

`kratos_add_dependency(${KRATOS_SOURCE_DIR}/applications/LinearSolversApplication)`